tester.sh

```diff
@@ -1,25 +1,55 @@
 #!/bin/bash
 
 RESET="\033[0m"
+BOLD="\033[1m"
+RED="\033[0;31m"
 V_GREEN="\e[0;38;5;82m"
 P_GREEN="\e[0;38;5;23m"
 
+usage()
+{
+	echo -e "usage:"
+	echo -e "bash tester.sh vector"
+	echo -e "bash tester.sh map"
+	exit
+}
+
+if (( $# != 1 )); then
+    usage
+fi
+
+ARG=$1
+if [ ${ARG} == "vector" ]; then
+    DIR=./tests/vector/
+elif [ ${ARG} == "map" ]; then
+    DIR=./tests/map/
+else
+	echo -e "\""$1"\" is not a valid argument."
+    usage
+fi
+
+if [ "$(eval uname)" == "Darwin" ]; then
+    DIFF=diff
+    V_GREEN="\033[32m"
+else
+    DIFF=diff --collor
```

Comment:
```suggestion
    DIFF="diff --color"
```

no linux (ao menos no wsl) precisa das aspas se não não seta a variável corretamente e dá xabu na execução 🙃